develop/dev-guide-sample-application-aws-lambda.md

```diff
@@ -0,0 +1,367 @@
+
+> **Note**
+>
+> For complete code snippets and running instructions, refer to the [tidb-samples/tidb-aws-lambda-quickstart](https://github.com/tidb-samples/tidb-aws-lambda-quickstart) GitHub repository.
+
+### Step 1: Clone the sample app repository
+
+Run the following commands in your terminal window to clone the sample code repository:
+
+```bash
+git clone git@github.com:tidb-samples/tidb-aws-lambda-quickstart.git
+cd tidb-aws-lambda-quickstart
+```
+
+### Step 2: Install dependencies
+
+Run the following command to install the required packages (including `mysql2`) for the sample app:
+
+```bash
+npm install
+```
+
+### Step 3: Configure connection information
+
+Connect to your TiDB cluster depending on the TiDB deployment option you've selected.
+
+<SimpleTab>
+
+<div label="TiDB Serverless">
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+
+2. Click **Connect** in the upper right corner. A connection dialog is displayed.
+
+3. Ensure the configurations in the connection dialog match your operating environment.
+
+    - **Endpoint Type** is set to `Public`
+    - **Connect With** is set to `General`
+    - **Operating System** matches your environment.
+
+    > **Note**
+    >
+    > In Node.js applications, you don't have to provide an SSL CA certificate, because Node.js uses the built-in [Mozilla CA certificate](https://wiki.mozilla.org/CA/Included_Certificates) by default when establishing the TLS (SSL) connection.
+
+4. Click **Create password** to create a random password.
+
+    > **Tip**
+    >
+    > If you have generated a password before, you can either use the original password or click **Reset password** to generate a new one.
+
+5. Copy and paste the corresponding connection string into `env.json`. The following is an example:
+
+    ```json
+    {
+      "Parameters": {
+        "TIDB_HOST": "{gateway-region}.aws.tidbcloud.com",
+        "TIDB_PORT": "4000",
+        "TIDB_USER": "{prefix}.root",
+        "TIDB_PASSWORD": "{password}"
+      }
+    }
+    ```
+
+    Replace the placeholders in `{}` with the values obtained in the connection dialog.
+
+</div>
+
+<div label="TiDB Self-Hosted">
+
+Copy and paste the corresponding connection string into `env.json`. The following is an example:
+
+```json
+{
+  "Parameters": {
+    "TIDB_HOST": "{tidb_server_host}",
+    "TIDB_PORT": "4000",
+    "TIDB_USER": "root",
+    "TIDB_PASSWORD": "{password}"
+  }
+}
+```
+
+Replace the placeholders in `{}` with the values obtained in the **Connect** window.
+
+</div>
+
+</SimpleTab>
+
+### Step 4: Run the code and check the result
+
+1. (Prerequisite) Install the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html).
+
+2. Build the bundle:
+
+    ```bash
+    npm run build
+    ```
+
+3. Invoke the sample Lambda function:
```

Comment:
**deploy lambda function** should be completed before invoking it or after?

Reply:
```suggestion
3. Invoke the sample Lambda function locally:
```